examples/plot_simulate_somato.py

```diff
@@ -94,43 +103,90 @@
+    weights_ampa=weights_ampa_p, weights_nmda=weights_nmda_p,
+    location='proximal', synaptic_delays=synaptic_delays_p, seedcore=6)
+
+# Late proximal drive
+weights_ampa_p = {'L2_basket': 0.003, 'L2_pyramidal': 0.0039,
+                  'L5_basket': 0.004, 'L5_pyramidal': 0.0020}
+weights_nmda_p = {'L2_basket': 0.001, 'L2_pyramidal': 0.0005,
+                  'L5_basket': 0.002, 'L5_pyramidal': 0.0020}
 synaptic_delays_p = {'L2_basket': 0.1, 'L2_pyramidal': 0.1,
                      'L5_basket': 1.0, 'L5_pyramidal': 1.0}
+
+net.add_evoked_drive(
+    'evprox2', mu=130.0, sigma=3., numspikes=1, sync_within_trial=True,
+    weights_ampa=weights_ampa_p, weights_nmda=weights_nmda_p,
+    location='proximal', synaptic_delays=synaptic_delays_p, seedcore=6)
+
+# Early distal drive
+weights_ampa_d = {'L2_basket': 0.0043, 'L2_pyramidal': 0.0032,
+                  'L5_pyramidal': 0.0009}
+weights_nmda_d = {'L2_basket': 0.0029, 'L2_pyramidal': 0.0051,
+                  'L5_pyramidal': 0.0010}
+synaptic_delays_d = {'L2_basket': 0.1, 'L2_pyramidal': 0.1,
+                     'L5_pyramidal': 0.1}
+
 net.add_evoked_drive(
-    'evprox1', mu=20.0, sigma=0., numspikes=1, weights_ampa=weights_ampa_p,
-    weights_nmda=weights_nmda_p, location='proximal',
-    synaptic_delays=synaptic_delays_p, seedcore=6)
+    'evdist1', mu=32., sigma=3., numspikes=1, sync_within_trial=True,
+    weights_ampa=weights_ampa_d, weights_nmda=weights_nmda_d,
+    location='distal', synaptic_delays=synaptic_delays_d, seedcore=6)
+
+# Late distal input
+weights_ampa_d = {'L2_basket': 0.0041, 'L2_pyramidal': 0.0019,
+                  'L5_pyramidal': 0.0018}
+weights_nmda_d = {'L2_basket': 0.0032, 'L2_pyramidal': 0.0018,
+                  'L5_pyramidal': 0.0017}
+synaptic_delays_d = {'L2_basket': 0.1, 'L2_pyramidal': 0.1,
+                     'L5_pyramidal': 0.1}
 
-dpl = simulate_dipole(net, n_trials=1)
+net.add_evoked_drive(
+    'evdist2', mu=82., sigma=3., numspikes=1, sync_within_trial=True,
+    weights_ampa=weights_ampa_d, weights_nmda=weights_nmda_d,
+    location='distal', synaptic_delays=synaptic_delays_d, seedcore=2)
 
-trial_idx = 0
-import matplotlib.pyplot as plt
-fig, axes = plt.subplots(2, 1, sharex=True, figsize=(6, 6))
-dpl[trial_idx].plot(ax=axes[0], show=False)
-net.cell_response.plot_spikes_hist(ax=axes[1])
-net.cell_response.plot_spikes_raster()
+###############################################################################
+# Now we run the simulation over 2 trials so that we can plot the average
+# aggregate dipole. As in :ref:`the MPIBackend example
+# <sphx_glr_auto_examples_plot_simulate_mpi_backend.py>`, we can use
+# ``MPIBackend`` to reduce the simulation time by parallizing across cells in
+# the network. However, no parallel backend is necessary. For a better
+# match to the empirical waveform, set ``n_trials`` to be >=25.
+n_trials = 2
+# n_trials = 25
+with MPIBackend(n_procs=6, mpi_cmd='mpiexec'):
```

Comment:
I think better to set this to `n_procs=1`? What would happen if some computer doesn't have 6 cores?

Reply:
Good point. How about `n_procs=2` so that the backend at least does something?

Reply:
+1 for `n_procs=2`, dual-core is standard since way back when.

FYI I ran the example on my dual-core i7 with `n_procs=6` and it worked without flaf! Of course each `nrniv` process was down to 33% as the CPU was getting choked on thread hopping :)

Reply:
But do we really need the explicit `mpi_cmd=`? I think it'll confuse newcomers more than it helps power-users.

Reply:
Sounds good. Updated in https://github.com/jonescompneurolab/hnn-core/pull/253/commits/3f665c45e7e76eb9d9282fc5ba025196f3368453.